docs/index.md

```diff
@@ -1379,31 +1379,31 @@ Value (string):
 ```
 OMA-URI:
 ```
-./Device/Vendor/MSFT/Policy/Config/Firefox~Policy~firefox~ContentAnalysis/ContentAnalysis_InterceptionPoints_Clipboard_Enabled
+./Device/Vendor/MSFT/Policy/Config/Firefox~Policy~firefox~ContentAnalysis~InterceptionPoints/ContentAnalysis_InterceptionPoints_Print
```

Comment:
issue: I think this should be `Clipboard` at the end? (instead of `Print`)

Reply:
You are correct sir. Fixed